File: openapi/code_samples/python/api_company_bankrupt_stats.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/top-10-companies-by-caen', methods=['GET'])
def find_top_10_companies_by_caen():
    caen = request.args.get('caen', '')
    year = int(request.args.get('year', 0))
    county = request.args.get('county', '')
    page_number = int(request.args.get('pageNumber', 0))
    page_size = int(request.args.get('pageSize', 20))
    sort_by = request.args.get('sortBy', 'caen')

    # Your implementation here

    return jsonify({}), 200

if __name__ == '__main__':
    app.run()

File: openapi/code_samples/csharp/api_company_bankrupt_stats.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("top-10-companies-by-caen")]
    [SwaggerOperation(Summary = "Retrieves the top 10 bankrupt company stats based on a specific CAEN.")]
    [ProducesResponseType(typeof(PageBankruptCompanyStatsView), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult FindTop10CompaniesByCaen(
        [FromQuery(Name = "caen")] string caen = "",
        [FromQuery(Name = "year")] int year = 0,
        [FromQuery(Name = "county")] string county = "",
        [FromQuery(Name = "pageNumber")] int pageNumber = 0,
        [FromQuery(Name = "pageSize")] int pageSize = 20,
        [FromQuery(Name = "sortBy")] string sortBy = "caen")
    {
        // Your implementation here
        return Ok();
    }
}
